education/education/doctype/fees/fees.json

```diff
@@ -365,11 +357,30 @@
    "fieldname": "print_settings_tab",
    "fieldtype": "Tab Break",
    "label": "Print Settings"
+  },
+  {
+   "allow_on_submit": 1,
+   "fetch_from": "student.student_email_id",
+   "fieldname": "student_email",
+   "fieldtype": "Data",
+   "label": "Student Email",
+   "options": "Email",
+   "print_hide": 1
+  },
+  {
+   "fetch_from": "student.student_email_id",
+   "fieldname": "contact_email",
```

Comment:
It's not ideal to have 2 fields storing the same data. student_email field should be renamed instead. Then write a patch that copies the data from the student_email field and pastes it into the new field. But it's a breaking change and can affect customizations of the community. Create a GitHub issue and see if anyone objects then go ahead with this change.

Reply:
Yes, I agree it's not an ideal way, but I thought that it may result in some issues, that is why I went with this approach. @pateljannat 

let me create an issue and see if anyone objects it.